### PR TITLE
Update even more enzyme tests

### PR DESCRIPTION
**Pull request by huwshimi**

### CODE REVIEW

**Comment by huwshimi** [comment]
[Update even more tests to use Enzyme](https://trello.com/c/8N2nWpUJ/390-update-even-more-tests-to-use-enzyme)

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-gce.jujugui.org:8080//job/juju-gui/8571/

File: jujugui/static/gui/src/app/components/button-dropdown/test-button-dropdown.js [hunk 1]
```diff
@@ -60,137 +55,72 @@ describe('Button Dropdown', function() {
...
-        </div>
-      );
-      expect(output).toEqualJSX(expected);
-      done();
-    });
+  it('can render open', () => {
+    const wrapper = renderComponent();
+    wrapper.find('.button-dropdown__button').simulate('click');
+    wrapper.update();
+    const menu = wrapper.find('WrappedDropdownMenu');
```
Comment (hatched) [inline jujugui/static/gui/src/app/components/button-dropdown/test-button-dropdown.js]: Oh nice, we dont need the component to check that it has been added.

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**